src/dateService.js

```diff
@@ -0,0 +1,18 @@
+import { differenceInCalendarDays, formatISO9075, subWeeks, startOfWeek, endOfWeek } from 'date-fns';
```

Comment:
**praise**: `date-fns` looks nice. I need to remember that one.

Reply:
i believe it's the recommended replacement for moment